test/template-tests/Format-AzureRMTemplate.ps1

```diff
@@ -23,6 +23,46 @@
             '$schema','contentVersion', 'apiProfile', 
             'parameters','functions','variables',
             'resources', 'outputs'
+
+        $resourceOrder = 'comments', 'condition', 'type', 'apiVersion', 'name', 
+            'location', 'sku', 'kind', 'dependsOn', 'tags', 'copy'
+
+        $sortProperties = {
```

Comment:
have we tested nested inline resources?

Reply:
At current this would only sort one level of resource depth.  You're right, this should be addressed.  Lmk if you'd prefer a rework of this function in this PR or a new one.